lighthouse-core/test/results/sample_v2.json

```diff
@@ -3543,51 +3529,51 @@
               "path": "3,HTML,1,BODY,36,A",
               "selector": "body > a",
               "nodeLabel": "external link",
-              "snippet": "<a href=\"https://www.google.com/\" target=\"_blank\">"
```

Comment:
also this is unexpected.  

i think we just need to manually update the AnchorElements bit in `lighthouse-core/test/results/artifacts/artifacts.json`

Reply:
> i think we just need to manually update the AnchorElements bit in `lighthouse-core/test/results/artifacts/artifacts.json`

`yarn update:sample-artifacts AnchorElements`, for reference

Reply:
you can specify the artifact ?!

Reply:
> you can specify the artifact ?!

haha, oh no...

I added it to the [contributing doc](https://github.com/GoogleChrome/lighthouse/blob/master/CONTRIBUTING.md#gatherer-prs)! I don't know where else we can put it to make it more prominent.